src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import duke.task.Todo;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.File;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test for Storage class used in Duke.
+ */
```

Comment:
I like that each test class only tests one thing at a time, in case one test fails, the rest of the tests can still run.